Second commit

naveengowdabh opened the pull request:
Hi
I have commit to Branch1,please review and approve.
regards
naveenbh629

naveengowdabh committed:
Second commit

naveengowda629 commented:
Hi naveenbh629,
your code looks good, please add one more check..
regards 
lead

naveengowdabh committed:
Hi
As per your previous comment I made changes in Hello class,please review
regards
naveenbh629

naveengowdabh commented:
As per your previous comment I made changes in Hello class, please review and approve
regards
naveenbh629

naveengowda629 commented:
code is looks perfect ,go merge it with master.
regards,
lead
